controllers/solrcloud_controller_test.go

```diff
@@ -88,6 +88,12 @@ var _ = FDescribe("SolrCloud controller - General", func() {
 						InitContainers:     extraContainers2,
 					},
 				},
+				Availability: solrv1beta1.SolrAvailabilityOptions{
```

Comment:
Since we want this enabled by default, I think we should remove this and test that the default does its job.

Or, keep this and add the `expectPodDisruptionBudget(ctx, solrCloud, solrCloud.StatefulSetName(), statefulSet.Spec.Selector, intstr.FromString(util.DefaultMaxPodsUnavailable))` line to another test (without removing the PDB, just do the existence check). Yeah I like this option more. Keep what you have and add the PDB check to another test in this file.